sql/user-account-management.md

```diff
@@ -0,0 +1,94 @@
+---
+title: TiDB 用户账户管理
+category: user guide
+---
+
+# TiDB 用户账户管理
+
+## 用户名和密码
+
+TiDB 将用户账户存储在 `mysql.user` 系统表里面。每个账户由用户名和 host 作为标识。每个账户可以设置一个密码。
+
+通过 MySQL 客户端连接到 TiDB 服务器，通过指定的账户和密码登陆：
+
+```
+shell> mysql --port 4000 --user xxx --password yyy
+```
+
+使用缩写的命令行参数则是：
+
+```
+shell> mysql -P 4000 -u xxx -pyyy
+```
+
+注意如果使用 `-p` 参数，那么 `-p` 和密码之间不能有空格。
+
+## 添加用户
+
+添加用户有两种方式：
+
+* 通过标准的用户管理的 SQL 语句创建用户以及授予权限，比如 `CREATE USER` 和 `GRANT` 。
+* 直接通过 `INSERT` `UPDATE` 和 `DELETE` 操作授权表。
+
+推荐的方式是使用第一种。第二种方式修改容易导致一些不完整的修改，因此不推荐。还有另一种可选方式是使用图形化界面工具。
+
+下面的例子用 `CREATE_USER` 和 `GRANT` 语句创建了四个账户：
```

Comment:
consistency. `CREATE_USER` for `CREATE USER`?
and line 58 uses `DROP USER` instead of `DROP_USER`